### ISSUE TITLE
Last commit error

### ISSUE THREAD

**Issue by tgensol** [arnesson/cordova-plugin-firebase#28]
Hi,

I tried to use the last commit and I got these errors : 

`
../platforms/android/src/org/apache/cordova/firebase/FirebasePlugin.java:218: error: unreported exception JSONException; must be caught or declared to be thrown
            String value = params.getString(key);
                                           ^
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error
`

Something is wrong in the new one right ?

**Comment by mmaloon** [comment]
I think it's just a matter of updating the logEvent method signature to throw JSONException:

```java
private void logEvent(final CallbackContext callbackContext, final String name, final JSONObject params) throws JSONException {
 ....
}
```

**Comment by remithomas** [comment]
Same issue here. Better to use the version 0.1.8

**Comment by robertarnesson** [comment]
fixed @f86f886ae629ea9e33e85b428795114f78b2b9a1
